npm/vue/src/index.ts

```diff
@@ -377,7 +430,10 @@ export function mount (componentOptions: any, options: any = {}) {
 
       // by returning undefined we keep the previous subject
       // which is the mounted component
-      return undefined
+      return {
```

Comment:
We can remove the `cy.wrap(wrapper, { log: false })` as it's redundant now

Reply:
done